spinalcordtoolbox/scripts/sct_deepseg_gm.py

```diff
@@ -69,20 +53,36 @@ def get_parser():
              "the following challenge: goo.gl/h4AVar.",
         choices=('large', 'challenge'),
         default='large')
-    misc.add_argument(
+    optional.add_argument(
         "-thr",
         type=float,
         help='Threshold to apply in the segmentation predictions, use 0 (zero) to disable it. Example: `0.999`',
         metavar=Metavar.float,
         default=0.999)
-    misc.add_argument(
```

Comment:
Nice!! I really like the idea of standardizing on QC == MISC, but all other (functional) optional args in OPTIONAL.

Reply:
That's very fair. I'm A-OK treating it as out of scope for now.